src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(100);
```

Comment:
Please extract Random to class variable as done in ColorSupplier
Please make all magic numbers as class constants with proper naming